Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM debian:stable-20170620
```

Comment:
Hello,

Is there a reason you are using the date tag and not just `debian:stable`?